dashboard/client/src/common/CodeDialogButton/CodeDialogButton.tsx

```diff
@@ -99,9 +101,10 @@ export const CodeDialogButtonWithPreview = ({
   code,
   buttonText,
   className,
+  sx,
   ...props
 }: CodeDialogButtonWithPreviewProps) => {
-  const classes = useCodeDialogButtonWithPreviewStyles();
+  const styles = useCodeDialogButtonWithPreviewStyles(useTheme());
```

Comment:
There shouldn't be a need to create an object like this.

When using sx props, you can also easily get access to the theme by passing a [function](https://mui.com/system/getting-started/the-sx-prop/#callback-values) into sx.

Reply:
In most cases, I would prefer if we inlined styles instead of creating a separate styles object, but this is not a blocker.

Reply:
Using inline styles is the right approach. I will try to fix them.

Reply:
Understood, I've removed the redundant object and directly passed the function instead. Additionally, converted some parts to use inlined styles, while leaving the rest as is for readability considerations.